ci/package.sh

```diff
@@ -68,47 +113,41 @@ do
+                            --build-arg PATCH_VERSION=$stack_version_patch \
+                            -t $DOCKERHUB_ORG/$stack_id \
                             -t $DOCKERHUB_ORG/$stack_id:$stack_version_major \
                             -t $DOCKERHUB_ORG/$stack_id:$stack_version_major.$stack_version_minor \
                             -t $DOCKERHUB_ORG/$stack_id:$stack_version_major.$stack_version_minor.$stack_version_patch \
                             -f $stack_dir/image/Dockerfile-stack $stack_dir/image
                     fi
-
-                    echo "  - id: $stack_id" >> $index_file_local
-                    sed 's/^/    /' $stack >> $index_file_local
-                    [ -n "$(tail -c1 $index_file_local)" ] && echo >> $index_file_local
-                    echo "    templates:" >> $index_file_local
                 else
                     echo -e "\n- SKIPPING stack: $repo_name/$stack_id"
                 fi
 
                 echo "  - id: $stack_id" >> $index_file
                 sed 's/^/    /' $stack >> $index_file
                 [ -n "$(tail -c1 $index_file)" ] && echo >> $index_file
-                echo "    templates:" >> $index_file
-
-                for template_dir in $stack_dir/templates/*/
-                do
-                    if [ -d $template_dir ]
-                    then
-                        template_id=$(basename $template_dir)
-                        template_archive=$repo_name.$stack_id.templates.$template_id.tar.gz
+                echo "    default-image: $DOCKERHUB_ORG/$stack_id:$stack_version" >> $index_file
 
-                        if [ $build = true ]
-                        then
-                            # build template archives
-                            tar -cz -f $assets_dir/$template_archive -C $template_dir .
-                            echo -e "--- Created template archive: $template_archive"
+                echo "  - id: $stack_id" >> $index_file_local
+                sed 's/^/    /' $stack >> $index_file_local
+                [ -n "$(tail -c1 $index_file_local)" ] && echo >> $index_file_local
+                echo "    default-image: $DOCKERHUB_ORG/$stack_id:$stack_version" >> $index_file_local
 
-                            echo "      - id: $template_id" >> $index_file_local
-                            echo "        url: file://$assets_dir/$template_archive" >> $index_file_local
-                        fi
-
-                        echo "      - id: $template_id" >> $index_file
-                        echo "        url: $RELEASE_URL/$stack_id-v$stack_version/$template_archive" >> $index_file
-                    fi
-                done
+                # Create or extract template archives
+                if [ -d $stack_dir/templates ]
+                then
+                    package_template "$stack_dir/templates" $build tar
+                elif [ -d $stack_dir/image/templates ]
+                then
```

Comment:
I'd like to avoid introducing lots of changes in one go.. so would like keep the templates location as is and do the rest. Can we update the PR to only deal with injecting meta info in the stack configs?